src/js/messaging/components/compose/MessageCategory.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ErrorableSelect from '../../../common/components/form-elements/ErrorableSelect';
+import ErrorableRadioButtons from '../../../common/components/form-elements/ErrorableRadioButtons';
+import { makeField } from '../../../common/model/fields.js';
+
+class MessageCategory extends React.Component {
+  render() {
+    const categories = this.props.categories;
+    const categoryValue = makeField(this.props.value);
+
+    return (
+      <div className="messaging-categories">
+        <fieldset className="messaging-categories--lg">
+          <legend className="usa-sr-only">Category</legend>
+          <ErrorableRadioButtons
+              label="Category: "
```

Comment:
Those projects also use block-layout for labels, so it's a moot question. This one does not. Whether it should is also a question for design.